tests/unit/bases/test_almalinux.py

```diff
@@ -811,19 +811,14 @@ def test_setup_snapd_proxy_failures(fake_process, fake_executor, fail_index):
 
 
 @pytest.mark.parametrize("fail_index", list(range(0, 2)))
-def test_pre_setup_snapd_failures(fake_process, fake_executor, fail_index):
+def test_pre_setup_snapd_failures(
+    mock_verify_network, fake_process, fake_executor, fail_index
+):
```

Comment:
As a preventative against linters later complaining about `mock_verify_network` being unused, you could do:

```suggestion
@pytest.mark.usefixtures("mock_verify_network")
def test_pre_setup_snapd_failures(fake_process, fake_executor, fail_index):
```

We currently have the annotation checks disabled in ruff for tests, which is [where ruff organises this check](https://beta.ruff.rs/docs/rules/unused-function-argument/). I don't really see a need to enable it for tests, but if we were to do so later this would flag.

Reply:
I like the preventative work, thanks!